### PR TITLE
[Fix]• Patch issues that prevent build on ccl

### PR DESCRIPTION
**Pull request by theangelperalta**

### CODE REVIEW

File: objc-runtime.lisp [hunk 1]
```diff
@@ -399,7 +399,7 @@
 (defmacro with-selectors ((&rest selector-specs) &body body)
   `(let (,@(mapcar (fw.lu:destructuring-lambda ((sym foreign-selector))
                      `(,sym (ensure-selector ,foreign-selector)))
-                   (mapcar (fw.lu:glambda (spec)
+                   (mapcar (fwoar.anonymous-gf:glambda (spec)
```
Comment (fiddlerwoaroof) [inline objc-runtime.lisp]: This change is already merged

File: bundle-utils.lisp [hunk 2]
```diff
@@ -30,6 +30,9 @@
           ("BUNDLE:SUPPORT;USER;**;*.*.*" ,(application-support-directory :user))
           ("BUNDLE:SUPPORT;LOCAL;**;*.*.*" ,(application-support-directory :local)))))
 
+;; FIXME: Compile time error with CCL setting up logical-pathname-translations
+;; Bundle
+#+(or)
```
Comment (fiddlerwoaroof) [inline bundle-utils.lisp:35]: I'd prefer a patch that fixes this functionality. Or, perhaps, splits bundle-utils into a separate system.

**Comment by fiddlerwoaroof** [comment]
I'm willing to take patches, but I'd like an explicit statement that I have permission to relicense contributed code: it's fairly unlikely, but I've occasionally thought about switching all of my projects to Apache v2

**Comment by theangelperalta** [comment]
> I'm willing to take patches, but I'd like an explicit statement that I have permission to relicense contributed code: it's fairly unlikely, but I've occasionally thought about switching all of my projects to Apache v2

You have my explicit permission to relicense my contributed code.

File: bundle-utils.lisp (hunk 2 above)
Comment (theangelperalta) [reply bundle-utils.lisp:35]: This evening I will investigate a solution that fixes this functionality. I'll pull the latest patches from `master` precede from there. Thank you for your feedback.

**Pull request closed without merging**